### DISCUSSION TITLE
Copilot not working in Shell script

### DISCUSSION THREAD

**Discussion by h4rvey-g** [community/community#70284] [Copilot Conversations]
### Select Topic Area

Bug

### Body

My copilot is not working in shell scripts, but it's working fine in other languages. The output shows something like this, with a blank for model deployment ID.
``` 
[INFO] [default] [2023-10-17T06:55:17.010Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-10-17T06:55:17.491Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 427 ms
[INFO] [streamChoices] [2023-10-17T06:55:17.540Z] request done: headerRequestId: [e339706e-4650-432f-87b6-316342ed0597] model deployment ID: []
```
Changing the language mode to other languages in .sh files would also fix it. What happened?

**Comment by sharpobject** [comment]
Hello, this is happening over here with .py files. Copilot can generate suggestions if I rename the file to .json or whatever.
